engine/pkg/promutil/util.go

```diff
@@ -77,12 +77,12 @@ func HTTPHandlerForMetric() http.Handler {
 //		$JobType_$Namespace_$Subsystem_$Name(actual)
 
 // NewFactory4Master return a Factory for jobmaster
-func NewFactory4Master(info tenant.ProjectInfo, jobType libModel.JobType, jobID libModel.MasterID) Factory {
```

Comment:
I think `MasterID` may be more proper here because this is the function for inner usage(master/worker abstract).
But the `jobType` make me struggle.

Reply:
> But the jobType make me struggle.

It's the same to me.